pallets/ah-migrator/src/staking/nom_pools.rs

```diff
@@ -141,3 +144,230 @@ impl<T: Config> Pallet<T> {
 		}
 	}
 }
+
+fn normalize_generic_bonded_pool<T: Config>(
+	mut pool: GenericBondedPoolInner<BalanceOf<T>, T::AccountId, BlockNumberFor<T>>,
+) -> GenericBondedPoolInner<BalanceOf<T>, T::AccountId, BlockNumberFor<T>> {
+	use crate::Pallet;
+
+	if let Some(ref mut throttle_from) = pool.commission.throttle_from {
+		*throttle_from = Pallet::<T>::rc_to_ah_timepoint(*throttle_from);
```

Comment:
not the best way to test it. ideally you would write your own map function to test the result of `rc_to_ah_timepoint`.
but I think we wont need the `rc_to_ah_timepoint`, since we should use RC block times when we get block provider types with 2503 sdk. so you can keep this as it is, just take in consideration for other tests.